packages/mobile/src/screens/notifications-screen/Notification/Tip.tsx

```diff
@@ -0,0 +1,18 @@
+import { formatNumberCommas } from 'audius-client/src/common/utils/formatUtil'
+
+import { AudioText } from 'app/components/core'
+
+import { NotificationText } from './NotificationText'
+
+type TipProps = {
+  value: number
+}
+
+export const Tip = (props: TipProps) => {
```

Comment:
Super nit: maybe call this `TipAmount`/`TipText`/`TipAmountText` or something less vague?

Reply:
Yeah agreed